### PR TITLE
Iss89

### LINKED ISSUE #89: Add DE function
- [x] Integrate DE function
- [ ] Add loop over fixed effects in vignette as an example

### PR DESCRIPTION
**Pull request by rvitancol**
Closes #89

### CODE REVIEW

**Comment by rvitancol** [comment]
Closing this, will create a new one without the norm code.

**Comment by NicoleEO** [comment]
Can you add NanoStringGeoMxSet-de.R to collate in description?

File: R/NanoStringGeoMxSet-de.R [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+#' target_demoDataSub <- target_demoDataSub[1:10, 1:10]
+#' mixedOutmc <- mixedModelDE(target_demoDataSub,
+#'                          elt = "q_norm",
+#'                          modelFormula = ~ pool_rep +  (1 | slide),
+#'                          groupVar = "pool_rep",
+#'                          nCores = 12,
+#'                          multiCore = TRUE,
+#'                          pAdjust = NULL)
+#'
+#' @export
```
Comment (NicoleEO) [inline R/NanoStringGeoMxSet-de.R:30]: Can you roxygenize or manual add to get these functions available outside the package?

File: tests/testthat/test_de.R [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+                           elt = "exprs_norm",
+                           modelFormula = NULL,
+                           groupVar = "pool_rep",
+                           nCores = 12,
+                           multiCore = TRUE,
+                           pAdjust = NULL
+)
+
+# Run parallel
+mixedOutp <- mixedModelDE(target_demoData,
```
Comment (NicoleEO) [inline tests/testthat/test_de.R:71]: I am getting this error:
image

**Comment by NicoleEO** [comment]
@rvitancol it it's too much of a hassle to resolve. Just copy the two files and update with my comments in a new branch off dev.

File: tests/testthat/test_de.R (hunk 2 above)
Comment (rvitancol) [reply tests/testthat/test_de.R:71]: this should be fixed now

**Comment by rvitancol** [comment]
@NicoleEO this is ready now.

**Review by NicoleEO — APPROVED** [review]

**Pull request merged**